test/core/TakerOperations.t.sol

```diff
@@ -197,54 +130,86 @@ contract TakerOperationsTest is MangroveTest {
     /* Setting fillWants = false means we should spend as little as possible to receive
        as much as possible.
        Here despite asking for .1eth the offer gives 1eth for ~0 so we should receive 1eth. */
-
-    Tick snipeTick = TickLib.tickFromPrice_e18(1 ether);
-    (uint successes, uint got, uint gave,,) = testMgv.snipesInTest(
-      $(base), $(quote), wrap_dynamic([ofr, uint(Tick.unwrap(snipeTick)), 0.01 ether, 100_000]), false
-    );
-    assertTrue(successes == 1, "Snipe should not fail");
+    Tick orderTick = TickLib.tickFromPrice_e18(1 ether);
+    (uint got, uint gave,,) = mgv.marketOrderByTick($(base), $(quote), Tick.unwrap(orderTick), 0.1 ether, false);
     assertApproxEqRel(got, 1 ether, relError(10), "Wrong got value");
     assertApproxEqRel(gave, 0.01 ether, relError(10), "Wrong gave value");
     assertTrue(!mgv.offers($(base), $(quote), ofr).isLive(), "Offer should not be in the book");
   }
 
-  function test_snipe_fillGives_zero() public {
```

Comment:
Similarly, shouldn't we test something similar with clean (ie that cleaning fails in this setup)?

Reply:
The test `test_cleaning_non_failing_offer_fails` tests the same, just in a slightly different way (it was written before migrating the snipes tests).